boards/nxp/frdm_rw612/doc/index.rst

```diff
@@ -0,0 +1,116 @@
+.. _frdm_rw612:
+
+NXP FRDM_RW612
+##############
+
+Overview
+********
+
+The RW612 is a highly integrated, low-power tri-radio wireless MCU with an
+integrated 260 MHz ARM Cortex-M33 MCU and Wi-Fi 6 + Bluetooth Low Energy (LE) 5.3 / 802.15.4
+radios designed for a broad array of applications, including connected smart home devices,
+gaming controllers, enterprise and industrial automation, smart accessories and smart energy.
+
+The RW612 MCU subsystem includes 1.2 MB of on-chip SRAM and a high-bandwidth Quad SPI interface
+with an on-the-fly decryption engine for securely accessing off-chip XIP flash.
+
+The advanced design of the RW612 delivers tight integration, low power and highly secure
+operation in a space- and cost-efficient wireless MCU requiring only a single 3.3 V power supply.
+
+Hardware
+********
+
+- 260 MHz ARM Cortex-M33, tri-radio cores for Wifi 6 + BLE 5.3 + 802.15.4
+- 1.2 MB on-chip SRAM
+
+Supported Features
+==================
+
++-----------+------------+-----------------------------------+
+| Interface | Controller | Driver/Component                  |
++===========+============+===================================+
+| NVIC      | on-chip    | nested vector interrupt controller|
++-----------+------------+-----------------------------------+
+| SYSTICK   | on-chip    | systick                           |
++-----------+------------+-----------------------------------+
+| MCI_IOMUX | on-chip    | pinmux                            |
++-----------+------------+-----------------------------------+
+| GPIO      | on-chip    | gpio                              |
++-----------+------------+-----------------------------------+
+| USART     | on-chip    | serial                            |
++-----------+------------+-----------------------------------+
+
+
+The default configuration can be found in the defconfig file:
+
+   :zephyr_file:`boards/nxp/frdm_rw612/frdm_rw612/`
```

Comment:
Path here needs to be updated

Reply:
i think this is a typo, this is a strange path